codecoverage.py

```diff
@@ -91,10 +91,15 @@ def generate_info(grcov_path):
         else:
             ordered_files.append("ccov-artifacts/" + fname)
 
+    # Assume we're on a one-click loaner.
+    mod_env = os.environ.copy()
+    if os.path.isdir('/home/worker/workspace/build/src/gcc/bin'):
```

Comment:
I found TASKCLUSTER_INTERACTIVE, which is only defined in one-click loaners. It's mentioned as a hack that could be removed at some point (https://jonasfj.dk/2016/03/one-click-loaners-with-taskcluster/), but it hasn't been removed in more than a year.